db/seeds.rb

```diff
@@ -3,6 +3,10 @@
 require 'csv'
```

Comment:
Changes to the seeds to allow for a reasonable initial data set for testing (office 99 goes to secondary queue, the rest go to primary)